lib/feature.pm

```diff
@@ -291,12 +291,6 @@ regardless of what feature declarations are in scope.
 
 =head2 The 'signatures' feature
 
-B<WARNING>: This feature is still experimental and the implementation may
-change or be removed in future versions of Perl.  For this reason, Perl will
-warn when you use the feature, unless you have explicitly disabled the warning:
-
-    no warnings "experimental::signatures";
-
 This enables unpacking of subroutine arguments into lexical variables
```

Comment:
I know you didn't write this, but this is still "programmer speak." I'd write:


```
This assigns subroutine arguments to lexical variables in the subroutine signature.
For example, for this subroutine:

    sub foo ($left, $right) {
        return $left + $right;
    }

Calling C<foo(3, 7);> will assign C<3> and C<7> to C<$left> and C<$right>, respectively,
in the subroutine.
```

Reply:
I'll give it an adjustment.